package-lock.json

```diff
@@ -27193,6 +27354,36 @@
         "vitest": "2.0.5"
       }
     },
+    "stake-token-prices/api": {
```

Comment:
I still see the old name being referenced here

Reply:
Now de diffs are huge. Review that! 😆

Reply:
Solved!

Reply:
> Now de diffs are huge. Review that! 😆

cmd + f and type `stake`, it was never so easy! 😆